### PR TITLE
update `StoryblokLink`

### PR DESCRIPTION
**Pull request by 21christiansc**

### CODE REVIEW

File: src/Api/ContentApi.php [hunk 1]
```diff
@@ -464,11 +464,14 @@ private function fetchDatasourceEntriesResultPage (
 	/**
 	 * Fetches all links.
 	 *
+	 * @param string|null $startsWith Can be used to get only the links whose full_slug starts with it.
```
Comment (keichinger) [inline src/Api/ContentApi.php]:
```suggestion
	 * @param string|null $startsWith Fetch only links where the `full_slug` matches the given pattern
```

File: src/Api/Data/StoryblokLink.php [hunk 2]
```diff
@@ -15,5 +18,15 @@ public function __construct (array $data)
 		$this->id = $data["id"];
 		$this->uuid = $data["uuid"];
 		$this->slug = $data["slug"];
+		$this->isFolder = $data["is_folder"];
```
Comment (keichinger) [inline src/Api/Data/StoryblokLink.php:21]: Proposal: Rename `isFolder` to `hasFolderTarget`, or is that too confusing or even too verbose?

File: src/Api/Data/StoryblokLink.php [hunk 3]
```diff
@@ -15,5 +18,15 @@ public function __construct (array $data)
...
+		$this->position = $data["position"];
+		$this->name = $data["name"];
```
Comment (keichinger) [inline src/Api/Data/StoryblokLink.php:23]: Proposal: Rename `name` to `label`, or is that confusing?

**Review by keichinger — DISMISSED** [review]

File: src/Api/Data/StoryblokLink.php (hunk 2 above)
Comment (21christiansc) [reply src/Api/Data/StoryblokLink.php:21]: We have decided to leave it at `isFolder`.

File: src/Api/Data/StoryblokLink.php (hunk 3 above)
Comment (21christiansc) [reply src/Api/Data/StoryblokLink.php:23]: We have decided to leave it at `name`.

File: src/Api/Data/StoryblokLink.php [hunk 4]
```diff
@@ -15,5 +18,15 @@ public function __construct (array $data)
...
+	}
+
+	/**
+	 */
```
Comment (apfelbox) [inline src/Api/Data/StoryblokLink.php:28]:
```suggestion
	 * @return string[]
	 */
```

**Review by apfelbox — COMMENTED** [review]
Apart from that last comment: 👍

**Pull request merged**